pkgs/applications/networking/mailreaders/alot/notmuch.nix

```diff
@@ -14,6 +15,21 @@ buildPythonPackage {
 
   buildInputs = [ python notmuch cffi ];
 
+  # since notmuch 0.35, this package expects _notmuch_config.py that is normally
+  # generated by notmuch's configure script
+  postPatch = let
+    versionFile = writeTextFile {
```

Comment:
I think it'd be better if we capture the file generated by the configure script as an additional output and consume that output here.